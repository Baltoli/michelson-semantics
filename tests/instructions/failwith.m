$contract { code { CAR ; NIL operation ; FAILWITH Unit ; PAIR }; 
storage unit ;
parameter unit ; } ; 
$paramtype unit ;
$storagetype unit ;
$balance 0 ;
$amount 0 ;
$now 0 ;
$myaddr 0 ;
$knownaddrs ;
$sourceaddr 0 ;
$senderaddr 0 ;
$param Unit ;
$storage Unit ;
