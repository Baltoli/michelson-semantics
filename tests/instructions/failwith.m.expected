<generatedTop>
  <k>
    Aborted ( "FAILWITH instruction reached" , Unit )
  </k>
  <paramtype>
    unit
  </paramtype>
  <storagetype>
    unit
  </storagetype>
  <stack>
    FailedStack
  </stack>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( 0 )
  </now>
  <myaddr>
    #Address ( 0 )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( 0 )
  </sourceaddr>
  <senderaddr>
    #Address ( 0 )
  </senderaddr>
</generatedTop>
